views/js/pciCreator/dev/spreadsheetsInteraction/runtime/spreadsheetsInteraction.amd.js

```diff
@@ -0,0 +1,166 @@
+define([
+    'IMSGlobal/jquery_2_1_1',
+    'qtiCustomInteractionContext',
+    'OAT/util/event',
+    '../lib/handsontable/handsontable'
```

Comment:
use the local pci referencer spreadsheetsInteraction/lib/handsontable/handsontable